src/configuration/services/mysql.md

```diff
@@ -79,6 +79,71 @@ foreach ($relationships['database'] as $endpoint) {
 2. MySQL Errors such as "PDO Exception 'MySQL server has gone away'" are usually simply the result of exhausting your existing diskspace. Be sure you have sufficient space allocated to the service in [.platform/services.yaml](/configuration/services.md).
 
 
+## Multiple databases
+
+On the 10.0 version and later of MariaDB it is possible to define multiple databases as well as multiple users with different permissions.  To do so requires defining multiple endpoints.  Under the `configuration` key of your service there are two additional keys:
```

Comment:
I reworded it to not say the name, since technically it's our service that only supports it as of 10.0.  MariaDB/MySQL itself has always supported multiple DBs, of course.